dali/python/nvidia/dali/ops.py

```diff
@@ -97,17 +97,43 @@ def _get_kwargs(schema):
     """
     ret = ""
     for arg in schema.GetArgumentNames():
-        allow_tensors = schema.IsTensorArgument(arg)
-        arg_name_doc = arg
-        dtype = schema.GetArgumentType(arg)
-        type_name = _type_name_convert_to_string(dtype, allow_tensors = allow_tensors)
-        if schema.IsArgumentOptional(arg):
-            type_name += ", optional"
-            if schema.HasArgumentDefaultValue(arg):
-                default_value_string = schema.GetArgumentDefaultValueString(arg)
-                default_value = ast.literal_eval(default_value_string)
-                type_name += ", default = {}".format(_default_converter(dtype, default_value))
-        doc = schema.GetArgumentDox(arg)
+        skip_full_doc = False
+        type_name = ""
+        dtype = None
+        doc = ""
+        deprecation_warning = None
+        if schema.IsDeprecatedArg(arg):
+            meta = schema.DeprecatedArgMeta(arg)
+            msg = meta['msg']
+            assert msg is not None
+            deprecation_warning = ".. warning::\n\n    " + msg.replace("\n", "\n    ")
+            renamed_arg = meta['renamed_to']
+            # Renamed and removed arguments won't show full documentation (only warning box)
+            skip_full_doc = renamed_arg or meta['removed']
+            # Renamed aliases don't are not fully registered to the schema, that's why we query for the
```

Comment:
```suggestion
            # Renamed aliases are not fully registered to the schema, that's why we query for the
```

Reply:
It's just a comment - we can skip CI if that's the going to be the only change in upcoming commit (just be sure not to `commit --amend`)